201-key-vault-secret-create/azuredeploy.json

```diff
@@ -54,10 +54,10 @@
             }
         },
         "secrets": {
-            "type": "array",
+            "type": "secureObject",
```

Comment:
Hi Brian, yeah I agree it makes more sense if we call the wrapper "secureObject" and the array in it "secrets". I have made these adjustments now.